applications/archive/archive-plugins/org.csstudio.archive.reader.appliance/src/org/csstudio/archive/reader/appliance/ApplianceValueIterator.java

```diff
@@ -320,4 +322,15 @@ protected static AlarmSeverity getSeverity(int severity) {
             return AlarmSeverity.UNDEFINED;
         }
     }
+
+    /**
+     * Determines alarm status from the given numerical representation.
+     *
+     * @param status numerical representation of alarm severity
```

Comment:
.. numerical representation of alarm _status_, not severity

Reply:
Corrected